docs/Reference/Reply.md

```diff
@@ -369,7 +369,7 @@ const serialize = reply
 serialize({ foo: 'bar' }) // '{"foo":"bar"}'
 ```
 
-See [.compileSerializationSchema(schema, httpStatus)](#compileserializationschema)
+See [.compileSerializationSchema(schema, [httpStatus])](#compileserializationschema)
```

Comment:
why this change?

Reply:
Just to indicate that the param is optional. Is that ok or do you think is better indicate it in some other way?